Assets/Scripts/SoundManager.cs

```diff
@@ -5,11 +5,11 @@ public class SoundManager : MonoBehaviour {
 
     //crincle sound found here: http://freesound.org/people/volivieri/sounds/37171/
 
-    //public AudioClip crincleAudioClip;
-    public AudioSource crincle;
+    public AudioClip crincleAudioClip;
+    AudioSource crincle;
 
 
-    /*void Awake()
+    void Awake()
```

Comment:
same goes for functions -- let's mark them explicitly `private` if they are intended to be so.